src/instruments/view/staffsettingsmodel.h

```diff
@@ -20,24 +20,22 @@
 #define MU_INSTRUMENTS_STAFFSETTINGSMODEL_H
 
 #include <QObject>
-#include <QQmlEngine>
 
 #include "modularity/ioc.h"
 #include "context/iglobalcontext.h"
 #include "notation/notationtypes.h"
 
-namespace mu {
-namespace instruments {
+namespace mu::instruments {
 class StaffSettingsModel : public QObject
 {
     Q_OBJECT
 
-    INJECT(instruments, context::IGlobalContext, globalContext)
+    INJECT(instruments, context::IGlobalContext, context)
 
-    Q_PROPERTY(QString staffType READ staffType NOTIFY staffTypeChanged)
-    Q_PROPERTY(QVariantList voices READ voices NOTIFY voicesChanged)
-    Q_PROPERTY(bool isSmallStaff READ isSmallStaff NOTIFY isSmallStaffChanged)
-    Q_PROPERTY(bool cutawayEnabled READ cutawayEnabled NOTIFY cutawayEnabledChanged)
+    Q_PROPERTY(QString staffType READ staffType NOTIFY dataChanged)
```

Comment:
please explain why you replaced with one common signal?

Reply:
One signal is easier to maintain. There is no need for a bunch of signals because the view for this model is very simple